### ISSUE TITLE
url redirection broken in dev version

### ISSUE THREAD

**Issue by jozydapozy** [slimphp/Slim#250]
When I do this:

``` php
$app->get('/contact/', function () {
    echo "contact-page";    
});
```

and go to http://example.com/contact (without trailing slash) I'm redirected to http://contact/ 
When I go to http://example.com/contact/ everything works as expected.

The stable version works as expected on both urls.

**Comment by codeguy** [comment]
Can you pull the latest changes from the `develop` or `release-1.6.0` branch? I am unable to duplicate this on my machine. I'm running this in Terminal:

```
curl -X GET http://slim/contact -v
```

And I see the expected HTTP response being a 301 redirect having this HTTP header

```
Location: /contact/
```

Let me know if you are still having issues and I'll continue to investigate.

**Comment by jozydapozy** [comment]
I don't use git, but when I download the latest dev-version (this zip-file: https://github.com/codeguy/Slim/zipball/develop) the problem still occurs.. 

This is my index.php:

``` php
<?php
    require 'C:/xampp/phplibs/slimdevnew/Slim.php';

    $app = new Slim(
        array(
            'debug' => false
        )
    );

    $app->get('/', function () {
        echo "index";
    });

    $app->get('/contact/', function () {
        echo "contact-page";    
    });

    $app->run();
```

and when I go http://localhost/contact (no slash!!) I'm redirected to http://contact/

**Comment by codeguy** [comment]
Are you on Windows? Please pull down the `release-1.6.1` branch and see if that fixes things.

Edit: Just saw the `require 'C:\xampp\...` path :) The `1.6.1` release should help you.

**Comment by jozydapozy** [comment]
You're right! Bug seems to be solved in `release-1.6.1` ! Great! Thank you!

**Comment by codeguy** [comment]
Awesome. That branch is stable if you want to keep using it. It should be merged into master this weekend.
